packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -286,7 +286,7 @@ class BackdropFilter extends SingleChildRenderObjectWidget {
 ///  * [Canvas], the class that a custom painter uses to paint.
 class CustomPaint extends SingleChildRenderObjectWidget {
   /// Creates a widget that delegates its painting.
-  const CustomPaint({ Key key, this.painter, this.foregroundPainter, this.size: Size.zero, Widget child })
+  const CustomPaint({ Key key, this.painter, this.foregroundPainter, this.size: Size.zero, this.forceRasterCache: false, Widget child })
     : assert(size != null),
```

Comment:
assert non-null. also, add a new paragraph to the constructor's docs saying that size and this bool must not be null.